test/Limiters/vertical_mass_borrowing_limiter.jl

```diff
@@ -0,0 +1,174 @@
+#=
+julia --project=.buildkite
+using Revise; include(joinpath("test", "Limiters", "vertical_mass_borrowing_limiter.jl"))
+=#
+using ClimaComms
+ClimaComms.@import_required_backends
+using ClimaCore: Fields, Spaces, Limiters
+using ClimaCore.RecursiveApply
+using ClimaCore.Geometry
+using ClimaCore.Grids
+using ClimaCore.CommonGrids
+import ClimaCore
+using Test
+using Random
+
+
+import Plots
+import ClimaCorePlots
+function plot_results(f, f₀)
+    col = Fields.ColumnIndex((1, 1), 1)
+    fcol = f[col]
+    f₀col = f₀[col]
+    p = Plots.plot()
+    Plots.plot(fcol; label = "field")
+    Plots.plot!(f₀col; label = "initial")
+    Plots.savefig("lim.png")
+end
+# plot_results(ρq, ρq_init)
```

Comment:
Could we enable a version of these plots either in the test artifacts or the documentation for this PR?